source/blog/2016-06-08-ember-2-6-and-2-7-beta-released.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Ember.js 2.6 and 2.7 Beta Released
+author: Matthew Beale
+tags: Releases
+---
+
+Ember.js 2.6, a minor version release of Ember with backwards compatible
+changes, is released today.
+
+Ember.js 2.7 beta, the branch of Ember that will be released as stable in
+roughly six weeks, is also being released today.
+
+### Changes in Ember.js 2.6
+
+No new features are added in Ember core in 2.6. In general the core team and
+community have remained active around other highly visible parts of the Ember
+stack (Ember Data, FastBoot, Glimmer, etc).
+
+#### Notable Deprecations
+
+The following deprecations are scheduled for release with Ember 2.6 and will be
+removed in Ember 3.0:
+
+* `Ember.String.htmlSafe(aString)` is the preferred API for marking a string
+  XSS safe for the rendering layer. Use of `Ember.Handlebars.SafeString` as
+  a constructor for safe strings is deprecated, and will be removed.
+  See the [deprecation guide](http://emberjs.com/deprecations/v2.x/#toc_use-ember-string-htmlsafe-over-ember-handlebars-safestring) for more details.
```

Comment:
We agreed @rwjblue will drop this line. Corresponding deprecation guide PR: https://github.com/emberjs/website/pull/2620